content/docs/cml/start-gitlab.md

```diff
@@ -0,0 +1,117 @@
+# Using CML on GitLab
+
+Here, we'll walk through a tutorial to start using CML on GitLab. For
+simplicity, we'll show the demo in GitLab Actions, but these instructions are
+valid for all supported Continuous Integration (CI) systems (with exceptions as
+noted!).
+
+1. Fork our
+   [example project repository](https://gitlab.com/iterative.ai/example_cml).
+   Click on Fork and select what namespace to fork the project to.
+
+   ![](/img/gitlab_fork_cml_project.png)
```

Comment:
BTW the image is great now but I just realized you can `<< Collapse sidebar` and then you get this view, which may be better for readability? 

![image](https://user-images.githubusercontent.com/1477535/108443285-24633980-721e-11eb-9027-299da40c5215.png)

You can also zoom in with the browser (Ctrl + `+`) until most of the width is used.